src/base/PhpBlock.php

```diff
@@ -67,7 +71,36 @@ public function renderAdmin()
         $this->injectorSetup();
         return $this->admin();
     }
-    
+
+    /**
+     * @inheritdoc
+     */
+    public function renderAdminPreview()
+    {
+        $image = $this->getPreviewImageSource();
+        if ($image) {
+            return Html::img($image);
+        }
+
+        return false;
+    }
+
+    /**
+     * Path to the preview image.
+     */
+    protected function getPreviewImageSource()
+    {
+        $imageName = $this->getViewFileName('jpg');
+        $imagePath = Yii::getAlias($this->ensureModule() . '/resources/img/' . $imageName);
```

Comment:
Maybe we could check for different paths like:

+ the name along with the block class (in the same folder of the block class)
+ the name along with the block view file (same folder of the view file)

As `resources/img` is only used for cms and admin module. For example there is no such folder in generic or bootstrap3 block extension (https://github.com/luyadev/luya-generic/tree/master/src)

What do you think?

Reply:
I dont like it to mix the block classes with images, same with view files. I think it no problem to make a extra folder for the images. Maybe it should be better `resources/block/` or `images/block/`?

Reply:
Ok i agree. What should we choice @luyadev/core `images/bock` or `resources/block`?

Reply:
@boehsermoe since there is no reply from other team members since 7 days we can do this the way we like (https://github.com/luyadev/luya/blob/master/docs/misc/organisation.md#7-rights-and-possibilities-of-members). I would go for `images/blocks`.